dataprofiler/tests/profilers/profiler_options/test_base_option.py

```diff
@@ -58,3 +62,12 @@ def test_eq(self):
         self.assertEqual(options, options)
         options2 = self.get_options()
         self.assertEqual(options, options2)
+
+    def test_json_decode(self):
+        expected_options = self.get_options()
+        if type(expected_options) == BaseOption:
```

Comment:
can we ignore test like this if `type()` is `BaseOption`?

Reply:
whats the actual issue, this seems problematic if that causes an issue

Reply:
if it doesn't error, then get rid of the skip.

Reply:
Otherwise, use mock for abstractmethods like we do for BaseProfiler